src/html.ml

```diff
@@ -128,9 +178,13 @@ and inline = function
   | Image (attr, { label; destination; title }) ->
       img label destination title attr
 
-let rec block = function
+let rec block ~auto_identifiers = function
```

Comment:
> I guess it means that we'll need to keep a state somewhere with all the ids we've already generated. Not sure how to best implement that and if that's something we want. Would love to have your input on that :)

Perhaps we can have the `block` function take a record for this argument that can carry some configuration/state? WDYT?

Reply:
Thanks for the suggestion, I added an `int StringMap.t` to keep track of the numerical suffix. Let me know what you think :)